src/index.js

```diff
@@ -27,6 +27,20 @@ const unexpectedEnzyme = {
       }
     );
 
+    childExpect.exportAssertion(
+      '<ReactWrapper> to render text <assertion>',
+      (expect, reactWrapper) => {
+        return expect.shift(reactWrapper.text());
+      }
+    );
+
+    childExpect.exportAssertion(
+      '<ReactWrapper> [not] to render text <string>',
+      (expect, reactWrapper, text) => {
+        expect(reactWrapper.text(), '[not] to equal', text);
+      }
```

Comment:
@sunesimonsen This is conflicting with the previous assertion, is there a way to handle that?

Reply:
The other one should be called `to render text satisfying`. The to satisfy behavior can be pretty surprising, so we usually mark it explicit.